src/components/PluginReadableInstalls.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 function PluginReadableInstalls({currentInstalls}) {
-    if (!currentInstalls) {
+    if (!currentInstalls && currentInstalls !== 0) {
```

Comment:
Does `!==` work as a comparison operator?  https://jsx.github.io/doc/operatorref.html does not mention it as a comparison operator
```suggestion
    if (!currentInstalls && currentInstalls != 0) {
```

Reply:
You should always use the triple version as it checks types and doesn't do boolean conversions (so false will never === 0 but will == 0)